### PR TITLE
make clusterizer kernels independent of grid size

### PR DESCRIPTION
**Pull request by VinInn**
as discussed in #564 and #586

### CODE REVIEW

File: RecoLocalTracker/SiPixelClusterizer/plugins/gpuClusterChargeCut.h [hunk 1]
```diff
@@ -19,105 +19,106 @@ namespace gpuClustering {
       uint32_t const* __restrict__ moduleId,     // module id of each module
       int32_t* __restrict__ clusterId,           // modified: cluster id of each pixel
       uint32_t numElements) {
-    if (blockIdx.x >= moduleStart[0])
```
Comment (VinInn) [inline RecoLocalTracker/SiPixelClusterizer/plugins/gpuClusterChargeCut.h:22]: very sorry. code-format has made the few (trivial) changes completely swamped by the new indentation

**Comment by fwyzard** [comment]
:construction: Validation running at fu-c2a02-35-01:/data/user/fwyzard/patatrack/validation/run_588.jqVUw5qfRF ...

File: RecoLocalTracker/SiPixelClusterizer/plugins/gpuClusterChargeCut.h (hunk 1 above)
Comment (fwyzard) [reply RecoLocalTracker/SiPixelClusterizer/plugins/gpuClusterChargeCut.h:22]: No problem - hiding the whitespace changes in git (`diff -w`) or in GitHub (gear icon --> `Hide whitespace changes`) alleviates the issue.

**Comment by fwyzard** [comment]
:construction: Validation running at fu-c2a02-35-01:/data/user/fwyzard/patatrack/validation/run_588.7Zsb7hPwxV ...

**Comment by fwyzard** [comment]
No impact on physics or throughput.

**Pull request merged**